relauncher.py

```diff
@@ -1,11 +1,61 @@
 import os, time
 
+# USER CHANGABLE ARGUMENTS
+
+# Change to `True` if you wish to enable these common arguments
+
+# Run upscaling models on the CPU
+extra_models_cpu = False
+
+# Automatically open a new browser window or tab on first launch
+inbrowser = False
+
+# Run Stable Diffusion in Optimized Mode - Only requires 4Gb of VRAM, but is significantly slower
+optimized = False
+
+# Run in Optimized Turbo Mode - Needs more VRAM than regular optimized mode, but is faster
+optimized_turbo = False
+
+# Creates a public xxxxx.gradio.app share link to allow others to use your interface (requires properly forwarded ports to work correctly)
+share = False
+
+
+# Enter other `--arguments` you wish to use - Must be entered as a `--argument ` syntax
+other_arguments = ""
+
+
+
+
+
+# BEGIN RELAUNCHER PYTHON CODE
+
+common_arguments = ""
+
+if extra_models_cpu == True:
+    common_arguments += "--extra-models-cpu "
+if optimized_turbo == True:
+    common_arguments += "--optimized-turbo "
+if optimized == True:
+    common_arguments += "--optimized "
+if share == True:
+    common_arguments += "--share "
+
+if inbrowser == True:

```

Comment:
`inbrowser` is confuing, maybe the argument should be "open_in_browser"?

Reply:
I can do that, I was trying to keep the naming convention aligned with what the parser arguments are, but I can clarify the arguments. Though, I already added comments saying what each of these does. Should it still be changed?

Reply:
I went ahead and renamed it. It does make it more understandable for a user.